include/memkind.h

```diff
@@ -407,6 +407,14 @@ void memkind_free(memkind_t kind, void *ptr);
 ///
 void *memkind_defrag_reallocate(memkind_t kind, void *ptr);
 
+///
+/// \brief Verify if path points to DAX-enabled file system
```

Comment:
Verifies if memkind temporary heap files in the ... directory can be created with the DAX attribute.

Reply:
I modify the comment but instead of "temporary heap files" I used "file-backed kind memory", let me know what you think

Reply:
it's fine.